web/assets/auto-apply-filter.js

```diff
@@ -0,0 +1,11 @@
+export default function autoApplyFilter() {
+    const filters = document.querySelectorAll("input.govuk-checkboxes__input");
```

Comment:
This should use a custom attribute for selection (e.g. `data-module="app-auto-apply-filter"`), otherwise it will be applied to any checkboxes in the application.